Add GetLabel method (#1057)
elastic/apm-agent-dotnet: docs/public-api.asciidoc

* `String key`:   The tag key
* `String|Number|bool value`: The tag value

[float]
[[api-transaction-get-label]]
==== `Label GetLabel(string key)` added[1.8.0,Number and boolean labels require APM Server 6.7+]

> Returns the label which was added in the <<api-transaction-set-label, SetLabel>> method. If the key does not exist, the method returns `null`.

Reviewer: ```suggestion
Returns the transaction's label. If the `key` does not exist, the method returns `null`.

TIP: Labels are added with the <<api-transaction-set-label, SetLabel>> method.
```

* `String key`:   The tag key
* `String|Number|bool value`: The tag value

[float]
[[api-span-get-label]]
==== `Label GetLabel(string key)` added[1.8.0,Number and boolean labels require APM Server 6.7+]

> Returns the label which was added in the <<api-span-set-label, SetLabel>> method. If the key does not exist, the method returns `null`.

Reviewer: ```suggestion
Returns the span's label. If the `key` does not exist, the method returns `null`.

TIP: Labels are added with the <<api-span-set-label, SetLabel>> method.
```
